src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-request.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: noticeNetworkRequest (iOS SDK API)
+type: apiDoc
+shortDescription: Manually record any transactional, HTTP-like network request that completes.
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - iOS SDK API
+metaDescription: New Relic API for iOS mobile app monitoring API to manually record any transactional, HTTP-like network request that completes.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-request-ios-sdk-api
+---
+
+## Syntax
+
+```
++ (void)noticeNetworkRequestForURL:(NSURL*)url
+                        httpMethod:(NSString*)httpMethod
+                         withTimer:(NRTimer*)timer
+                   responseHeaders:(NSDictionary*)headers
+                        statusCode:(NSInteger)httpStatusCode
+                         bytesSent:(NSUInteger)bytesSent
+                     bytesReceived:(NSUInteger)bytesReceived
+                      responseData:(NSData *)responseData
+                      traceHeaders:(NSDictionary<NSString*,NSString*>* _Nullable)traceHeaders
+                         andParams:(NSDictionary * _Nullable)params;
+```
+
+Manually record any transactional, HTTP-like network request that completes.
```

Comment:
@cdillard-NewRelic What does "HTTP-like" mean? I think it needs clarity if it includes anything more than a regular HTTP network request.

Reply:
I changed this to the more specific but still accurate: 
`Manually record any transactional HTTP network request that completes.`